### PR TITLE
Code Review

### PR DESCRIPTION
**Pull request by peterdtitan**
Adding a new branch for admissions trail code.

### CODE REVIEW

**Comment by peterdtitan** [comment]
Hi @peterdtitan , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that 

- [ ] doesn't have a descriptive title and short summary that tells in more detail what you've done in PR
- [ ] doesn't have linters set up correctly please try to fix that  

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @peterdtitan, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️  Good design for review summary



# Required Changes ♻️

- [x] Kindly make sure to use the inline comment in your review, this is a requirement.
- [x] Kindly always make sure to indicate the reason why the change is requested, this will help in not only making the developer write better code but acquire more knowledge.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        <h1 id="title">Nelson Mandela</h1>
+        <p>In honor of a Great Man...</p>
+      </div>
+    
+      <div id="img-div">
+        <img id="image" 
+        src="https://static.euronews.com/articles/338985/1024x538_338985.jpg" 
+        alt="Nelson Mandela's image">
+        <figcaption id="img-caption">The imagery of an immortal leader with multi-generational impact for all men in all places</figcaption>
+    </div>
```
Comment (peterdtitan) [inline index.html:23]: - [ ] Please consider using proper indentations on your tags. This gives your code a clean look. Fixing the Linters would help highlight this.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
```
Comment (peterdtitan) [inline index.html:60]: _Optional_
- [ ]  You might want to remove the comments to avoid having bogus files.

File: README.md [hunk 3]
```diff
@@ -1 +1,4 @@
-# nedramrez_admission_trail
+# Acknowledgments
+
+This code belongs to Zermarden@gmail.com. All rights reserved.
+> Purely copied and licensed for the purpose of performing a code review.
```
Comment (peterdtitan) [inline README.md:4]: - [ ] Please consider using the Microverse README.md template provided as it is more descriptive.

**Review by peterdtitan — COMMENTED** [review]
Hi @peterdtitan , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code due to the following reasons.

## Invalid Code Review Request

You have submitted a project, that 

- [ ] doesn't have a descriptive title and short summary that tells in more detail what you've done in PR
- [ ] doesn't have linters set up correctly please try to fix that 
- [ ] Has styling errors.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Also you may want to view the required changes with your work below:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by jaflih — APPROVED** [review]
Hi @peterdtitan   , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 




## Highlights
Linters working properly ✔️
Commit messages are meaningful ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**